plugins/scripting/scripting-ide-services/src/org/jetbrains/kotlin/scripting/ide_services/compiler/compiler.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.scripting.ide_services.compiler
+
+import org.jetbrains.kotlin.cli.common.messages.AnalyzerWithCompilerReport
+import org.jetbrains.kotlin.scripting.compiler.plugin.impl.*
+import kotlin.script.experimental.api.*
+import kotlin.script.experimental.host.ScriptingHostConfiguration
+import kotlin.script.experimental.api.ReplCompletionResult
+import kotlin.script.experimental.jvm.defaultJvmScriptingHostConfiguration
+import kotlin.script.experimental.util.calcAbsolute
+
+class KJvmReplCompilerImpl(hostConfiguration: ScriptingHostConfiguration = defaultJvmScriptingHostConfiguration) :
```

Comment:
The name could be confused with the other `KJvmReplCompilerImpl` that you've removed from the jvm-host. I'd suggest to rename it a bit.

Reply:
Fixed -> `KJvmReplIdeCompilerImpl`